components/IndicatorStat/utils/data.ts

```diff
@@ -12,21 +12,21 @@ const getLocalValue = (data: LocationData, options: ValueOptions): string => {
   if (data.meta) {
     const meta: LocationDataMeta = JSON.parse(data.meta);
     if (meta.valueLocalCurrency) {
-      return addPrefixAndSuffix(formatNumber(meta.valueLocalCurrency), options);
+      return addPrefixAndSuffix(formatNumber(meta.valueLocalCurrency, 1), options);
     }
   }
 
-  return addPrefixAndSuffix(formatNumber(data.value), {
+  return addPrefixAndSuffix(formatNumber(data.value, 1), {
     ...options,
     prefix: options.dataFormat === 'currency' ? 'US$' : options.prefix
   });
 };
 
-const getValue = (data: LocationData, options: ValueOptions): string => {
+const getValue = (data: LocationData, options: ValueOptions, decimalCount: number): string => {
   if (options.useLocalValue) {
     return getLocalValue(data, options);
   }
-  return addPrefixAndSuffix(formatNumber(data.value), options);
+  return addPrefixAndSuffix(formatNumber(data.value, decimalCount), options);
```

Comment:
I guess this will become `options.decimalCount`, for if you've build the CMS side configuration.

Reply:
Updated